### DISCUSSION TITLE
Praise and suggestion

### DISCUSSION THREAD

**Discussion by musicgreator** [apohl79/audiogridder#481] [Ideas]
I'm still in the testing stage of AG before I seriously implement it into my workflow. But I tried using it locally by searching for 127.0.0.1 in the plugin. The result is, that AG is handling realtime audio better than Reaper, without dropouts, perhaps because of higher buffers or better CPU handling. It would be cool to be able to switch instances or full sessions between different servers  without loosing the loaded plugins and their settings, even cross platform/ cross plugin format, where it has the option to look for the same plugin name in a different format. 

And even better would be, if it was possible to drag and drop local plugins from the dawn into AG and then back into the DAW, so that at any stage of the work process, one can decide to offload local plugins from the DAW into AG and back.

**Comment by apohl79** [comment]
Thanks for this. I understand your asks, but most of these things are not easy to be implemented via the current plugin frameworks.

> It would be cool to be able to switch instances or full sessions between different servers without loosing the loaded plugins and their settings

this is possible if you have the used plugins on both servers

> even cross platform/ cross plugin format, where it has the option to look for the same plugin name in a different format.

this will not be reliable as names are not sufficient and can be conflicting. plugins have unique id's, but unfortunately some vendors assign different id's for different formats or OS platforms. 

> And even better would be, if it was possible to drag and drop local plugins from the dawn into AG and then back into the DAW, so that at any stage of the work process, one can decide to offload local plugins from the DAW into AG and back.

this would be cool indeed... but no way I'm afraid.
